handbook/engineering/security/goals.md

```diff
@@ -2,38 +2,51 @@
+3. TODO: Do we need additional logs for cloud GA? I would argue [GSuite log ingestion](https://github.com/sourcegraph/sourcegraph/issues/17235) and [org level GCP audit logs](https://github.com/sourcegraph/sourcegraph/issues/17335)
+4. We have confidence that our centralized logging is production-ready, and we have relevant tooling and testing environments configured.
+5. **Sourcegraph Cloud is Generally Available for user to add private code**
+6. We're confident that we're ingesting logs from all services that are tightly coupled to our Cloud deployment. This is anything that directly interacts with our cloud deployment with elevated privileges, or directly allows authentication to these services.
+   1. For example, the Cloud deployment & GCP project, GCloud, GitHub, Buildkite, deployment pipelines, Cloudflare, Elasticsearch, the sourcegraph GCP organization, GCP groups, remote terraform state, etc.
+   2. We should also audit the logs that we currently ingest to ensure they contain sufficient information. If they don't, we need to increase their scope.
+7. We're ingesting logs from all services loosely coupled to our Cloud deployment. This is anything that cannot independently impact Cloud, but may be leveraged to gain access, extract sensitive information, or otherwise negatively impact cloud.
+   1. For example, non-production GCP projects, Slack, honeycomb, Zoom, etc.
+8. Normalize log format in our centralized log storage so that it is easier to correlate and search. This is a prerequisite for creating automated alerts from the logs.
+9. Create alerts and dashboards to automate the process of investigating events of interest (e.g., detect and alert on a spike of failed login attempts to a single account, or across Sourcegraph Cloud as a whole).
+10. We run a time-bound capture the flag event where there are larger bounties for being able to gain access to our [test security repository](https://github.com/sourcegraph/security-test/blob/main/README.md) on Sourcegraph Cloud.
+11. We connect our [test security repository](https://github.com/sourcegraph/security-test/blob/main/README.md) to Sourcegraph Cloud and only [members who can access that repository on GitHub](https://github.com/sourcegraph/security-test/settings/access) can access that repository on Sourcegraph Cloud (i.e., [Sourcegraph organization owners](https://github.com/orgs/sourcegraph/people?query=role%3Aowner) and [@sourcegraph/security](https://github.com/orgs/sourcegraph/teams/security) members). We then advertise a bounty for each unique vulnerability that allows an unauthorized person to gain access to this [test security repository](https://github.com/sourcegraph/security-test/blob/main/README.md) on Sourcegraph Cloud.
+12. We host our private [infrastructure repository](http://github.com/sourcegraph/infrastructure) on Sourcegraph Cloud.
+13. Document and publish our security practices so that our customers can 
```

Comment:
can what? 😅

Reply:
Thanks for catching that, I must have seen a crow outside the window and gotten distracted.